### DISCUSSION TITLE
powered_by

### DISCUSSION THREAD

**Discussion by giscus** [devlive-community/datacap#218] [Q&A] [answered]
# powered_by

DataCap is integrated software for data transformation, integration and visualization.

https://datacap.edurt.io/powered_by.html

**Comment by qianmoQ** [comment, marked as answer]
If you need our assistance to add, please reply in the following format:

`Your avatar`: xxxxx
`Your name`: xxxxx
`Type`: Personal |Project|Company
`Your Description` : xxxxx
`Your personal connection`:  xxxxx

> All the above `xxxxx` need to be replaced with your actual information. After we review it, we will add the information you provided to the site.

> If it is a business, we recommend that you convert the link address to your business address.
